From 4b6e5c74218af11d4480789c1a1302e1f0b692e3 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 26 Apr 2023 23:16:05 +0100
Subject: [PATCH 06/14] gnu: pytest-filter-subpackage: Update to 0.1.2.

* gnu/packages/python-check.scm (pytest-filter-subpackage): Update to
0.1.2.
[build-system]: Use pyproject-build-system.
[arguments]: Drop no more required phases.
[native-inputs]: Add python-setuptools-scm.
---
 gnu/packages/python-check.scm | 22 +++++++---------------
 1 file changed, 7 insertions(+), 15 deletions(-)

diff --git a/gnu/packages/python-check.scm b/gnu/packages/python-check.scm
index dd0600a2fc..3be8920e3c 100644
--- a/gnu/packages/python-check.scm
+++ b/gnu/packages/python-check.scm
@@ -633,27 +633,19 @@ (define-public python-pytest-exploratory
 (define-public python-pytest-filter-subpackage
   (package
     (name "python-pytest-filter-subpackage")
-    (version "0.1.1")
+    (version "0.1.2")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "pytest-filter-subpackage" version))
        (sha256
-        (base32 "1s4s2kd31yc65rfvl4xhy8xx806xhy59kc7668h6b6wq88xgrn5p"))))
-    (build-system python-build-system)
-    (arguments
-     '(;; One test is failing. There's an issue reported upstream. See
-       ;; https://github.com/astropy/pytest-filter-subpackage/issues/3.
-       ;; Disable it for now.
-       #:phases
-       (modify-phases %standard-phases
-         (replace 'check
-           (lambda* (#:key inputs outputs #:allow-other-keys)
-             ;; Make the installed plugin discoverable by Pytest.
-             (add-installed-pythonpath inputs outputs)
-             (invoke "pytest" "-vv" "-k" "not test_with_rst"))))))
+        (base32 "10hpl3f7g2bm29lakmp8492b7lr0dp90khfni12m4gl02xks7bhz"))))
+    (build-system pyproject-build-system)
     (native-inputs
-     (list python-pytest python-pytest-cov python-pytest-doctestplus))
+     (list python-pytest
+           python-pytest-cov
+           python-pytest-doctestplus
+           python-setuptools-scm))
     (home-page "https://github.com/astropy/pytest-filter-subpackage")
     (synopsis "Pytest plugin for filtering based on sub-packages")
     (description
-- 
2.39.2

